test/Confluent.Kafka.IntegrationTests/Tests/Consumer_OffsetsForTimes.cs

```diff
@@ -38,63 +38,66 @@ public void Consumer_OffsetsForTimes(string bootstrapServers)
             const int N = 10;
             const int Partition = 0;
 
-            var messages = ProduceMessages(bootstrapServers, singlePartitionTopic, Partition, N);
-
             var consumerConfig = new ConsumerConfig
             {
                 GroupId = Guid.NewGuid().ToString(),
                 BootstrapServers = bootstrapServers
             };
 
-            var firstMessage = messages[0];
-            var lastMessage = messages[N - 1];
-            using (var consumer = new ConsumerBuilder<byte[], byte[]>(consumerConfig).Build())
+            using (var topic = new TemporaryTopic(bootstrapServers, 1))
```

Comment:
This change is simply to use a temporary topic instead of the (old, to be discontinued) pre-created single partition topic. There was a bug in the test that assumed the topic was empty which I didn't pick up on previously because i didn't run the complete set of tests.